src/common/components/Textarea/Textarea.theme.ts

```diff
@@ -0,0 +1,39 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type TextareaVariants = VariantProps<typeof textareaTheme>;
+
+export const textareaTheme = tv(
+  {
+    slots: {
+      wrapper: [],
+      textarea: [
+        "flex",
+        "items-start",
+        "p-2",
+        "w-80",
+        "max-w-[30rem]",
+        "min-h-32",
+        "gap-2",
+        "rounded-lg",
+        "border",
+        "border-solid",
+        "bg-surface-base",
+        "border-border-default",
+        "text-text-em-mid",
+      ],
+    },
+    variants: {
+      size: {
+        md: {
+          wrapper: [],
+          textarea: ["text-sm"],
+        },
+        sm: {
+          wrapper: [],
+          textarea: ["text-xs"],

```

Comment:
I think we can follow input's text size for this!

```suggestion
          wrapper: [],
          textarea: ["text-base"],
        },
        sm: {
          wrapper: [],
          textarea: ["text-sm"],

```

Reply:
updated at [7e7b5b2](https://github.com/AfterClass-io/afterclass.io-v2/pull/82/commits/7e7b5b2379c272ba5c2b127f5154146ed291fd57)!